connector_helpscout/models/helpscout_mailbox_folder/importer.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import (
+    mapping,
+    only_create,
+)
+
+
+class HelpScoutMailboxFolderImportMapper(Component):
+    _name = 'helpscout.import.mapper.mailbox.folder'
+    _inherit = 'helpscout.import.mapper'
+    _apply_on = 'helpscout.mailbox.folder'
+
+    direct = [('type', 'helpscout_type'),
+              ('modified_at', 'backend_date_modified'),
+              ]
+
+    def _get_user_name(self, user_id):
+        user = self.env['helpscout.user'].search([
+            ('external_id', '=', user_id),
+        ]).odoo_id
+        return user.name
+
+    def _internal_name(self, record):
+        if record.type == 'mine':
+            user_name = self._get_user_name(record.user_id)
+            return '%s (%s)' % (record.name, user_name)
+        return record.name
+
+    @mapping
+    @only_create
+    def id(self, record):
```

Comment:
Hrmmmm so this won't work the way you expect. If we need to map multiple Helpscout objects to one Odoo object, we'll need to use the `_inherits` strategy and make this `odoo_id`.

Side note - never try to manipulate `id`. It's an auto-increment, unique, and managed by the DB. You can change it, but that will cause major issues.

Reply:
Ah, thanks. Would `_inherits` work in this case though, since the binding is it's own odoo model? Would a computed `odoo_id` field based on the id work instead?

Reply:
Wait, nevermind, the computed field idea was bad.

Reply:
You have to make a model that is just for the binding. https://github.com/LasLabs/odoo-connector-easypost/blob/f2dfbccdc6a7930c68f6d67749101d8eeeb8f87c/connector_easypost/models/address/common.py#L47